pkg/stanza/fileconsumer/file.go

```diff
@@ -199,22 +199,22 @@ func (m *Manager) makeFingerprint(path string) (*fingerprint.Fingerprint, *os.Fi
 	}
 	file, err := os.Open(path) // #nosec - operator must read in files defined by user
 	if err != nil {
-		m.Debugf("Failed to open file", zap.Error(err))
+		m.Errorw("Failed to open file", zap.Error(err))
 		return nil, nil
 	}
 
 	fp, err := m.readerFactory.newFingerprint(file)
 	if err != nil {
 		if err = file.Close(); err != nil {
-			m.Errorf("problem closing file %s", file.Name())
+			m.Errorw("problem closing file", zap.Error(err))
```

Comment:
Does this error include the file name?

Reply:
for reference: https://cs.opensource.google/go/go/+/master:src/os/file_unix.go;l=319;drc=a278550c40ef3f01a5fcbef43414dc49009201f8?q=openFileNolog&ss=go%2Fgo

Reply:
it returns something like:

close FILE_NAME: file already closed

Reply:
> It only returns error when it's already closed

I think this is why it was moved to debug level. Why is this important enough to warrant an error?

(Separately, I will be circling back to focus on readers again soon, how we manage them, etc. I'm intending to find a design that naturally avoids us trying to re-close a file.)

Reply:
I haven't changed level for this log, it was onl
I have only changed log level for https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/26706/files#diff-2aae1a01d477f0eef24857efb66f15c81cdacb550a8fe95c73b97e5ed3a6f0e4R202

Reply:
the reason for changing this log is that it didn't print the `err`, which it should in ideal case.

Reply:
Fair, but while we're evaluating levels. Should we lower this? It seems inactionable to the user.

Reply:
Sure, we can. I get your point. We should probably move it to debug because the only case in which it returns err is "file already closed".
Seems inactionable to user, as you said.